src/app/core/header/navbar/navbar.component.ts

```diff
@@ -41,12 +44,17 @@ export class NavbarComponent implements OnInit {
     });
 
     this.dialogWithForm.afterClosed()
+    .pipe(takeUntil(this.destroy$))
```

Comment:
format code in the next manner: 

`.pipe(
operator1(),
operator2()
)`

Reply:
Done)